config/development.json

```diff
@@ -6,11 +6,11 @@
     "port": 2023
   },
   "database": {
-    "connectionUri": "http://arangodb:8529",
+    "connectionUri": "http://127.0.0.1:8529",
```

Comment:
this will break the docker container

Reply:
I will create a production json config then.